### PR TITLE
[50_3] reduce the size of draw.data by remove plugins

### PR DESCRIPTION
**Pull request by Oyyko**
## Why you open this Pull Request?

reduce the size of draw_wasm by remove plugins

## What work have you done in the current Pull Request?

- [x] remove plugins in wasm data file.

### CODE REVIEW

File: xmake/draw.lua [hunk 1]
```diff
@@ -72,7 +72,9 @@ function add_target_draw()
     add_files("src/Mogan/Draw/draw.cpp")
 
     if is_plat("wasm") then
-        add_ldflags("-s --preload-file $(projectdir)/TeXmacs@TeXmacs", {force = true})
+        add_ldflags("-s --preload-file $(projectdir)/TeXmacs/fonts@TeXmacs/fonts", {force = true})
+        add_ldflags("-s --preload-file $(projectdir)/TeXmacs/packages@TeXmacs/packages", {force = true})
+        add_ldflags("-s --preload-file $(projectdir)/TeXmacs/...@TeXmacs/...", {force = true})
```
Comment (darcy-shen) [inline xmake/draw.lua:77]: ... 这个目录并不存在。

--preload-file 是指定生成的 draw.data 里面的数据以及映射，之前是将 TeXmacs目录的所有文件都放进去，实际上，至少 TeXmacs/plugins 是不需要的

Comment (darcy-shen) [reply xmake/draw.lua:77]: 比如 TeXmacs本来有7个目录，你只要写6个目录即可，移除掉不需要的`TeXmacs/plugins`目录

Comment (Oyyko) [reply xmake/draw.lua:77]: Got it

**Comment by Oyyko** [comment]
现在移除了plugins和tests

**Comment by darcy-shen** [comment]
It is about 49m before.

**Comment by darcy-shen** [comment]
Awesome work, it is 23.3M now!

**Pull request merged**